stage1/init/kvm/resources.go

```diff
@@ -66,3 +66,18 @@ func GetAppsResources(apps schema.AppList) (totalCpus, totalMem int64) {
 
 	return totalCpus, totalMem
 }
+
+// GetAppCapabilities is a filter which returns list of valid Linux capabilities
+// Function requires a manifest of application
+func GetAppCapabilities(app *types.App) []types.LinuxCapability {
+
+	var caps []types.LinuxCapability
+
+	for _, isolator := range app.Isolators {
+		switch capSet := isolator.Value().(type) {
```

Comment:
if there is only one case, `if` could be better solution than `switch `- a few lines less